docs/assets/js/main.js

```diff
@@ -75,9 +75,95 @@ function injectFooterCopywrite() {
     footer.innerHTML = `&copy;2008-${new Date().getFullYear()} Expensify, Inc.`;
 }
 
+function openSidebar() {
+    document.getElementById('sidebar-layer').style.display = 'block';
+
+    // Make body unscrollable
+    const yAxis = document.documentElement.style.getPropertyValue('y-axis');
+    const body = document.body;
+    body.style.position = 'fixed';
+    body.style.top = `-${yAxis}`;
+}
+
+// Function to close the sidebar
+function closeSidebar() {
+    document.getElementById('sidebar-layer').style.display = 'none';
+
+    // Make the body scrollable again
+    const body = document.body;
+    const scrollY = body.style.top;
+
+    // Reset the position and top styles of the body element
+    body.style.position = '';
+    body.style.top = '';
+
+    // Scroll to the original scroll position
+    window.scrollTo(0, parseInt(scrollY || '0', 10) * -1);
+}
+
+// Function to change SVG viewBox from Google
+function changeSVGViewBoxGoogle() {
+    // Get all inline Google SVG elements on the page
+    const svgsGoogle = document.querySelectorAll('svg');
+
+    // Create a media query for screens wider than tablet
+    const mediaQuery = window.matchMedia('(min-width: 800px)');
+
+    // Check if the viewport is smaller than tablet
+    if (!mediaQuery.matches) {
+        // Iterate through each SVG element
+        Array.from(svgsGoogle).forEach((svg) => {
+            // Set the viewBox attribute to '0 0 13 13' to make the svg fit in the mobile view
+            svg.setAttribute('viewBox', '0 0 13 13');
+            svg.setAttribute('height', '13');
+            svg.setAttribute('width', '13');
+        });
+    } else {
+        // Iterate through each SVG element
+        Array.from(svgsGoogle).forEach((svg) => {
+            // Set the viewBox attribute to '0 0 20 20' to make the svg fit in the tablet-desktop view
+            svg.setAttribute('viewBox', '0 0 20 20');
+            svg.setAttribute('height', '16');
+            svg.setAttribute('width', '16');
+        });
+    }
+}
+
+// Function to insert element after another
```

Comment:
Explain instead why we're insert an element after another

Reply:
Done

Reply:
Done